gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/resources/schema/schema-form.json

```diff
@@ -1,27 +1,91 @@
+            "description": "The list of headers to add to the request to the callback URL."
+        },
+        "authentication": {
+            "type": "object",
+            "id": "urn:jsonschema:io:gravitee:plugin:entrypoint:webhook:configuration:Authentication",
+            "title": "The authentication method to reach the callback URL.",
+            "description": "The authentication method to reach the callback URL.",
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "title": "The authentication type.",
+                    "description": "The authentication type.",
+                    "enum": ["BASIC", "BEARER"]
+                },
+                "jwksEndpoint": {
+                    "type": "string",
+                    "title": "The JWKS endpoint to use to retrieve the public key to verify the JWT signature.",
+                    "description": "The JWKS endpoint to use to retrieve the public key to verify the JWT signature."
+                },
+                "tokenEndpoint": {
+                    "type": "string",
+                    "title": "The token endpoint to use to retrieve the access token.",
+                    "description": "The token endpoint to use to retrieve the access token."
+                },
+                "clientId": {
+                    "type": "string",
+                    "title": "The client ID to use to retrieve the access token.",
+                    "description": "The client ID to use to retrieve the access token."
+                },
+                "clientSecret": {
+                    "type": "string",
+                    "title": "The client secret to use to retrieve the access token.",
+                    "description": "The client secret to use to retrieve the access token."
+                },
+                "scopes": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "title": "The scopes to use to retrieve the access token.",
+                    "description": "The scopes to use to retrieve the access token."
+                },
+                "secret": {
+                    "type": "string",
+                    "title": "The secret to use to verify the JWT signature.",
+                    "description": "The secret to use to verify the JWT signature."
+                },
+                "username": {
+                    "type": "string",
+                    "title": "The username to use to authenticate to the callback URL.",
+                    "description": "The username to use to authenticate to the callback URL."
+                },
+                "password": {
+                    "type": "string",
+                    "title": "The password to use to authenticate to the callback URL.",
+                    "description": "The password to use to authenticate to the callback URL."
```

Comment:
What are all these new configurations ? Not sure they are implemented already (or if they will) ?

Reply:
I've just read the configuration class: https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/java/io/gravitee/plugin/entrypoint/webhook/configuration/WebhookEntrypointConnectorConfiguration.java

Reply:
For the value of "type" I've just let github copilot fill the blank for me ^^

Reply:
Ok, I didn't see it. As security seems not yet implemented I would suggest to remove it from the schema because I'm not sure we will handle all these authentication modes. WDYT ?